examples/terraform-aws-s3-example/main.tf

```diff
@@ -17,6 +17,12 @@ resource "aws_s3_bucket" "test_bucket" {
   }
 }
 
+resource "aws_s3_bucket_policy" "bucket_access_policy" {
+  count = "${var.with_policy == "true" ? 1 : 0}"
```

Comment:
If you switch `with_policy` to bool, this also needs to be updated:

```
${var.with_policy ? 1 : 0}
```

Reply:
Implemented